nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/script/ScriptingComponentUtils.java

```diff
@@ -64,5 +68,15 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
+
+    /** A property descriptor for specifying whether to use dynamic relationships or not */
+    public static final PropertyDescriptor USE_DYNAMIC_RELATIONSHIPS = new PropertyDescriptor.Builder()
+            .name("Use Dynamic Relationships")
+            .description("Whether properties prefixed with \"REL_\" should create a dynamic relationship or not. The default will change to \"true\" in NIFI 2.0.")
+            .required(false)
```

Comment:
The convention for properties that use AllowableValues is to make them `required(true)`, not a big deal here since it will always be populated, but I mention it for consistency with other processors/properties.

Reply:
done